fl4health/clients/basic_client.py

```diff
@@ -391,23 +391,23 @@ def train_step(self, input: torch.Tensor, target: torch.Tensor) -> Tuple[Losses,
             target (torch.Tensor): The target corresponding to the input.
 
         Returns:
-            Tuple[Losses, Dict[str, torch.Tensor]]: The losses object from the train step along with
+            Tuple[TrainingLosses, Dict[str, torch.Tensor]]: The losses object from the train step along with
                 a dictionary of any predictions produced by the model.
         """
         # Clear gradients from optimizer if they exist
         self.optimizers["global"].zero_grad()
 
         # Call user defined methods to get predictions and compute loss
         preds, features = self.predict(input)
-        losses = self.compute_loss(preds, features, target)
+        losses = self.compute_training_loss(preds, features, target)
 
-        # Compute backward pass and update paramters with optimizer
+        # Compute backward pass and update parameters with optimizer
```

Comment:
Nice! 😂

Reply:
Any spelling errors in the repo we can assume came from 😭